source-map.bs

```diff
@@ -2,16 +2,21 @@
 Title: Source Map
 H1: Source Map
 Shortname: source-map
-Level: 1
+Level: none
 Status: STAGE0
 URL: https://tc39.es/source-map/
+Repository: https://github.com/tc39/source-map/
+Issue Tracking: GitHub Issues https://github.com/tc39/source-map/issues
+Test Suite: https://github.com/tc39/source-map-tests
+Editor Term: Editor, Editors
 Editor: Asumu Takikawa, Igalia
 Former Editor: Victor Porof, Google
 Former Editor: John Lenz, Google
 Former Editor: Nick Fitzgerald, Mozilla
 Previous Version: https://docs.google.com/document/d/1U1RGAehQwRypUTovF1KRlpiOFze0b-_2gc6fAH0KY0k/edit?pli=1#
 Repository: source-map/source-map-spec
-Abstract: A specification for mapping transpiled source code back to the original sources.  This specification is a living document and describes a hardened version of the Source Map v3 specification.
+Abstract: This Ecma standard defines the Source Map format, used for mapping transpiled source code back to the original sources.
+No Abstract: true
```

Comment:
Just unfamiliar with this, why no abstract is set to true when we now have an abstract?

Reply:
Because in that PDF with rules about not having a standard it says to not have an "Abstract", and instead a "Scope" and an "Introduction". I left a short abstract because I _think_ bikeshed can use it in some cases, as metadata to describe what the spec is about.